src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/ButtonAccessibility.m

```diff
@@ -40,17 +40,7 @@ - (nullable NSString *)accessibilityLabel
 
 - (BOOL)accessibilityPerformPress
 {
-    AWT_ASSERT_APPKIT_THREAD;
```

Comment:
I guess you need to remove the static "jclass sjc_CAccessibility = NULL;" above as well?

Reply:
And unneeded imports too. Remnants of the code that is gone. Fixed.